processing/topics/roads_bikelanes/bikelanes/BikelaneCategories.lua

```diff
@@ -2,7 +2,7 @@ package.path = package.path .. ";/processing/topics/roads_bikelanes/bikelanes/ca
 package.path = package.path .. ";/processing/topics/helper/?.lua"
 require("IsTermInString")
 require("IsSidepath")
-require("AddAdjoiningOrIsolated")
+require("AdjoiningOrIsolatedWrapper")
```

Comment:
Naming… 🤔 
Wrapper in der React-Welt packt etwas drumrum. Das hier splittet aber etwas auf aus eines in drei.
=> Was fällt uns hier besseres ein?

Reply:
Vielleicht "`CreateSubcategoriesAdjoiningOrIsolated`"?